Component/Product/Image.php

```diff
@@ -90,19 +90,19 @@ public function isValueURL($url)
     public function downloadFile($value)
     {
         /**
-         * @var ZendClient $client
+         * @var Client $client
          */
-        $client = $this->httpClientFactory->create();
+        $client = $this->clientFactory->create(['config' => [
+            'base_uri' => $value
+        ]]);
         $response = '';
 
         try {
-            $response = $client
```

Comment:
@benjamenhogben I have changed to use Guzzle library as per your recommendations. Fell free to comment to add some subtleties that I may not have thought of. thanks